packages/scss/demo/components/forms/forms-framed.html

```diff
@@ -0,0 +1,186 @@
+<section class="contentSection">
+	<h2>Form mod-framed</h2>
+	<p>
+		Form has can have a mod called <code class="code">.mod-framed</code>.
+		In this mod <code class="code">.form-group</code> behave differently and you have access to a new
+		DOM level called <code class="code">.form-group-line</code>.
+	</p>
+	<p>
+		<code class="code">form-group</code> represents a whole section of your form.
+		It can have a <code class="code">.form-group-title</code> to name your section
+		It can contain <code class="code">.form-group-line</code> and <code class="code">.input.mod-framed</code>
+	</p>
+	<p>
+		In a <code class="code">form-group-line</code> you can use <code class="code">.grid-*</code> classes onto your <code class="code">.input</code> to size
```

Comment:
grid- are supposed to have a grid parent, so I'm not sure that it's a clean way to do it.
Furthermore, as grids are not used here as containers, it may be possible to break framed form by updating grid component.
So what about using form framed relative mod classes? (Something like mod-framed1/2/3 with flex: 1/2/3 attributes).

Reply:
I didn't want to duplicate the grid system elsewhere that's why I went this way. But has it is a lightweight system with only 2-3 options, it might be worth a shot.